crates/reporting/tests/test_reporting.rs

```diff
@@ -10782,7 +10782,7 @@ All branches in an `if` must have the same type!
     `main` is defined directly in terms of itself:
 
     3│>  main =
-    4│>      if Bool.true then \{} -> {} else main
+    4│>      if Bool.true then {} else main
 
     Since Roc evaluates values strict, running this program would create
     an infinite number of `main` values!
```

Comment:
not a super big fan of the phrasing here. I'd tighten it up as 

> Roc evaluates values strictly, so running this program would enter an infinite loop!

or maybe 

> Roc evaluates values strictly, so running this program would enter an infinite loop trying to determine the value of `main`!

not super relevant for this PR, but maybe something to look into. @rtfeldman thoughts?

Reply:
I've applied your first suggestion, Folkert.